src/frontend/komponenter/Behandling/Vurdering/VisEllerEndreVurdering.tsx

```diff
@@ -60,7 +59,11 @@ const VisEllerEndreVurdering: FC<Props> = ({
             );
         case Redigeringsmodus.VISNING:
             return (
-                <VisVurdering vurdering={vurdering} settRedigeringsmodus={settRedigeringsmodus} />
+                <VisVurdering
+                    vurdering={vurdering}
+                    settRedigeringsmodus={settRedigeringsmodus}
+                    lagreVurdering={lagreVurdering}
```

Comment:
Trenger vi kanskje å sende med feilmelding hit også - hvis "nullstill" ikke går bra?

Reply:
Hmm, altså, hvis kallet failer, så ikke resett, men heller vis feilmelding?

Reply:
Like this?` 7edf77f`